quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -138,16 +139,35 @@ impl DefaultDocMapperBuilder {
 
         // Build the root mapping entry, it has an empty name so that we don't prefix all
         // field name with it.
-        let field_mappings = FieldMappingEntry::root(FieldMappingType::Object(self.field_mappings));
+        let root_field_mapping =
+            FieldMappingEntry::root(FieldMappingType::Object(QuickwitObjectOptions {
+                field_mappings: self.field_mappings,
+            }));
+
+        let field_entries = root_field_mapping.compute_field_entries();
+        let required_fast_fields: Vec<FieldPath> = field_entries
+            .into_iter()
+            .filter(|(_, field_type)| match field_type {
```

Comment:
the current tantivy master has `field_type.is_fast`, but this version is not yet used in quickwit

Reply:
oh nice!